features/collection.feature

```diff
@@ -0,0 +1,11 @@
+Feature:
+  In order to use postman with Api Platform
```

Comment:
`I should be able to generate a Postman collection file` should be enough, don't you think?